src/lib/services/datasets.ts

```diff
@@ -5,10 +5,6 @@ export async function createDatasetName(queryId: string, name: string): Promise<
   if (name === '' && name.trim() === '') {
     throw 'Please input a Dataset ID name';
   }
```

Comment:
I think the other validation should get moved back here or this should get moved as well. Seeing that this datasets.ts could be used elsewhere I think it should be here. (We have talked above allowing them to save the dataset without packaging the data)